settings.go

```diff
@@ -22,35 +33,70 @@ func Enabled() (map[string]bool, bool) {
 	keys := strings.Split(val, ",")
 
 	for _, key := range keys {
-		all[key] = true
+		// Trim in case people pass space-gapped names.
+		all[strings.TrimSpace(key)] = true
 	}
 
 	return all, false
 }
 
+// IsEnabled - Returns whether a given logger is present and enabled.
 func IsEnabled(name string) bool {
 	if allEnabled {
 		return true
 	}
 
-	_, ok := enabled[name]
-	return ok
+	value, ok := enabled[name]
+	if !ok {
+		return false
+	}
+	return value
 }
 
-func Verbosity() int {
-	level := os.Getenv("LOG_LEVEL")
+// SetEnabled - Enable or Disable loggers by name.
+func SetEnabled(name string, state bool) {
+	enabled[name] = state
+}
 
-	if strings.ToUpper(level) == "TIMER" {
-		return 2
-	}
+// SetAllEnabled - Enable or Disable *all* loggers.
+// NB: This does not change the status of individual loggers, only whether *all*
+// loggers should be allowed print. In other words, setting this to 'false' does
+// not disable loggers that are permitted to print, only those that are not.
+func SetAllEnabled(state bool) {
+	allEnabled = state
+}
 
-	if strings.ToUpper(level) == "ERROR" {
+// Populates default value of 'verbosity' variable from Envvar.
+func initVerbosity() int {
+	switch strings.ToUpper(os.Getenv("LOG_LEVEL")) {
+	case "ERROR":
 		return 3
+	case "TIMER":
+		return 2
+	default:
+		return 1
 	}
+}
 
-	return 1
+// Verbosity - Get the value of the LOG_LEVEL environment variable, converted to an integer.
+// Default/Invalid value is 1. If "TIMER", this is 2. If "ERROR", this is 3.
+func Verbosity() int {
```

Comment:
I think we can remove this, too, by just making `verbosity` variable capital case (public)